### PR TITLE
[#12464] Add verification step for Docker instance of Datastore in docs

### LINKED ISSUE #12464: Add verification step for Docker instance of Datastore in docs
The [docs](https://teammates.github.io/teammates/setting-up.html) on running datastore as a Docker instance does not have any verification step similar to the one for Solr
```
Verification: the Solr admin console should be accessible in http://localhost:8983.
```

The user should also be able to verify at `http://localhost:8484/` that the datastore instance is up
image

### PR DESCRIPTION
**Pull request by NicolasCwy**
Fixes #12464

**Outline of Solution**

Add verification step to docs, tested on local Markbind server
image

### CODE REVIEW

**Comment by wkurniawan07** [comment]
I'm not sure how useful this is going to be, given we're going to move out of Datastore soon anyway.
Also, it should be port 8484?

**Comment by NicolasCwy** [comment]
@wkurniawan07 I see. In any case, I've amended the docs, my mistake

File: docs/development.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ We have a Docker compose definition to run dependent services, including local D
 docker-compose run -p 8484:8484 datastore
 ```
 
+**Verification:** Should receive an "OK" response in the browser at `http://localhost:8484`.
```
Comment (domlimm) [inline docs/development.md:213]: Just a nit 🤣 
```suggestion
**Verification:** Should receive an "Ok" response in the browser at `http://localhost:8484`.
```

**Review by domlimm — APPROVED** [review]
Even though added a nitpick, but LGTM!

**Comment by NicolasCwy** [comment]
@domlimm Hi have resolved your suggestions, any thing I can do to move this forward?

**Comment by domlimm** [comment]
@NicolasCwy Thanks! Let's get another review.

**Pull request merged**